### PR TITLE
Refactor by using an array of directives

### LINKED ISSUE #38: Use a loop when constructing the security.txt file
In this set of lines:

https://github.com/lirantal/express-security-txt/blob/f48bfe2a56ce880eeec805a9240a0b354a7113f7/index.js#L41-L63

We could probably make the code shorter/easier to modify if we had an array or object and then looped over it. I'm thinking either we have an object like this, to map our notation to the policy's notation:

```
{
  contact: "Contact",
  encryption: "Encryption",
  ...
}
```

Or, we have an array and a function:

```
function capitalised(text) {
  return "...";
}

const fields = ["contact", "encryption", "..."]
```

We could then modify the existing loop at https://github.com/lirantal/express-security-txt/blob/master/index.js#L66-L74 to go through every known key we support, and capitalise it (either with an object or function) and then check if the value was provided. If it wasn't we drop it _in the loop_

I think this would make adding new properties faster. We could also have the schema validation use our array/object when deciding which policy fields are allowed?

### PR DESCRIPTION
**Pull request by joker314**
Resolves #38 

The array of directives is at the top, and can easily be modified if any new directives are added.

### CODE REVIEW

**Comment by joker314** [comment]
Of course, one still needs to specify what you're allowed to put inside a directive in the `validate` method.
<hr>

I add a (public) method `camelCase`. Should I make it more private? If so, how?

**Comment by joker314** [comment]
Lint is failing due to a security issue. It claims that `a[b]` notation is bad: but it appears to be a false positive, since `b` in our case is one of the values in the array at the top of the file (i.e., we have full control over it) so it can't hijack the prototype chain or anything like that.

Additionally, we do a `hasOwnProperty` check, so `a[b]` will not be accessing the prototype at all.

I'm going to disable the lint for that line or find a 'safer' way of writing it down.

File: index.js [hunk 1]
```diff
@@ -105,6 +87,20 @@ class middleware {
 
     return true
   }
+
+  /**
+   * Converts a security.txt directive like 'Contact'
+   * to the camelCase field name like 'contact'
+   *
+   * We assume that the contains a sequence of capitalised
```
Comment (lirantal) [inline index.js]: "We assume that the contains" ?

**Comment by lirantal** [comment]
LGTM

**Comment by joker314** [comment]
Nice catch! I fixed the typo; but for me to be allowed to merge it I need you to rereview.

**Comment by joker314** [comment]
Also: increased coverage to 100%. Ready for re-review.

**Review by lirantal — APPROVED** [review]

**Pull request merged**